website/content/en/releases/14.3R/hardware.adoc

```diff
@@ -2528,6 +2528,17 @@ sysctl(8).
 
 
 
+The
+**ure**
+device driver provides support for the following Ethernet controllers:
```

Comment:
So, a practical example of why information duplication is to be avoided whenever possible as a cornerstone of information architecture, look at this text.
```suggestion
driver supports the following USB Ethernet controllers:
```

We lost a critical detail writing this here that makes this section useful. Check out the justification I wrote in `man style.mdoc` for more details.

Reply:
Question. Would it be better to swap that part out with `&hwlist.ure;`?

Reply:
It will get swapped out very soon with that actually no matter what we put.